data/src/main/java/com/nimbl3/data/service/response/ErrorResponse.kt

```diff
@@ -0,0 +1,8 @@
+package com.nimbl3.data.service.response
+
+import com.google.gson.annotations.SerializedName
+
+data class ErrorResponse(
+    @SerializedName("message")
```

Comment:
Optional: redundant

Reply:
@Tuubz there are 2 approach when working with json objects definition:
- only define `SerializedName` for mismatched property names
- define all `SerializedName` for any property names

From my experiences, to avoid mistake, we should take care of defining `SerializedName` for all property names because it would bring a consistent field definitions code format, yeah?
```
data class ErrorResponse(
    @SerializedName("message")
    val message: String,
    @SerializedName("error_code")
    val errorCode: String
)
```
vs
```
data class ErrorResponse(
    val message: String,
    @SerializedName("error_code")
    val errorCode: String
)
```

Reply:
I love consistency 😍 🆗

Reply:
How about this?
```java;
data class ErrorResponse(
    @Transient
    val message: String,
    @SerializedName("error_code")
    val errorCode: String
)
```

https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.jvm/-transient/

Reply:
@lydiasama we're not discussing how we ignore a field out of a json object (with Transient), instead should we keep defining `@SerializedName` for the same field name and json key name.